### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by rehandalal**
phew!

r?

### CODE REVIEW

**Comment by rehandalal** [comment]
CI issue is:
https://github.com/less/less.js/issues/3295#issuecomment-407102613

File: package.json [hunk 1]
```diff
@@ -41,63 +42,65 @@
...
-    "less": "3.7.1",
+    "less": "3.8.0",
     "less-loader": "4.1.0",
     "nsp": "3.2.1",
     "object-assign": "4.1.1",
-    "postcss-flexbugs-fixes": "3.2.0",
-    "postcss-loader": "2.0.8",
+    "postcss-flexbugs-fixes": "4.0.0",
+    "postcss-loader": "2.1.6",
     "prettier": "1.13.7",
```
Comment (magopian) [inline package.json]: Just FYI prettier is now up to 1.14.0 according to https://github.com/mozilla/delivery-console/pull/324

File: .circleci/config.yml [hunk 2]
```diff
@@ -23,23 +23,23 @@ jobs:
       # Download and cache dependencies
       - restore_cache:
           keys:
-          - v1-dependencies-{{ checksum "package.json" }}
+          - v2-dependencies-{{ checksum "package.json" }}
```
[the remark this answers is not in the document]
Comment (rehandalal) [reply .circleci/config.yml]: It was not, hence we ran into CI issues related to:
https://github.com/less/less.js/issues/3295#issuecomment-407102613

File: package.json (hunk 1 above)
Comment (rehandalal) [reply package.json]: yeah I left that one open so that we could upgrade there

Comment (rehandalal) [reply package.json]: i mostly wanted to get webpack and the other packages that were exposed by ejecting upgraded.

**Comment by rehandalal** [comment]
bors r=@magopian

**Pull request merged**